telemetry/roles/prometheus_gaudi/tasks/deploy_gaudi_metric_exporter.yml

```diff
@@ -0,0 +1,66 @@
+# Copyright 2024 Intel Corporation.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+---
+
+
+# Assuming Prometheus was deployed and name space monitoring was created
+
+- name: Define base URL variable
+  set_fact:
+    base_url: "https://vault.habana.ai/artifactory/gaudi-metric-exporter/yaml/"
+    ds_name: "/metric-exporter-daemonset.yaml"
+    svc_name: "/metric-exporter-service.yaml"
+    sm_name: "/metric-exporter-serviceMonitor.yaml"
+
+- name: Define full URL
+  set_fact:
+    ds_url: >
+      {{ base_url }}{{ metric_exporter_version }}{{ ds_name }}
+    svc_url: >
+      {{ base_url }}{{ metric_exporter_version }}{{ svc_name }}
+    sm_url: >
+      {{ base_url }}{{ metric_exporter_version }}{{ sm_name }}
+
+- name: Deploy Gaudi metric exporter daemonset
```

Comment:
As discussed with Adam please fix the bug which was raised on gaudi3 metrics exporter.

Reply:
let's keep this PR on hold until we get the bug fixed

Reply:
As agreed, This has been tested on Gaudi2 let's merge this with the known bug on Gaudi3

Reply:
This is fixed on Gaudi3 now